2023/spring/Scheffler-MW@0230PM/Alina-Hordiienko.md

```diff
@@ -0,0 +1,4 @@
+<h1>Hi! I'm Alina.</h1>
+<h3>I was born and raised in Kyiv, Ukraine and lived all over the globe since. </h3>
+<p>nowadays, I'm based in **Tbilisi, Georgia** and **Cambridge, UK** - and will be in **Hyderabad, India** for the next 3 months.</p>
```

Comment:
Hey there! Here I am not sure if the bold text is being displayed correctly -> when I visualize it the text is still with ** (maybe taking <p> out would work :D